tensorboard/plugins/distribution/distributions_plugin.py

```diff
@@ -74,7 +74,11 @@ def frontend_metadata(self):
     )
 
   def distributions_impl(self, tag, run):
-    """Result of the form `(body, mime_type)`, or `ValueError`."""
+    """Result of the form `(body, mime_type)`.
+
+    Raises:
+      tensorboard.errors.PublicError: On invalid request.
```

Comment:
Not important: the error message will say histogram. Can that be confusing? FWIW, I am fine as is.

Reply:
Correct, and good point. It might be somewhat confusing, but it could
also be confusing the other way (there _aren’t_ any separate tags for
distributions). It’s also not a regression, and `http_api.md` has
likewise used the same wording, so it’s fine with me, too.